v4/export/config.go

```diff
@@ -668,10 +673,32 @@ func adjustConfig(conf *Config, fns ...func(*Config) error) error {
 
 func registerTLSConfig(conf *Config) error {
 	if len(conf.Security.CAPath) > 0 {
-		tlsConfig, err := utils.ToTLSConfig(conf.Security.CAPath, conf.Security.CertPath, conf.Security.KeyPath)
+		var err error
+		var tlsConfig *tls.Config
+		if len(conf.Security.SSLCABytes) == 0 {
+			conf.Security.SSLCABytes, err = ioutil.ReadFile(conf.Security.CAPath)
+			if err != nil {
+				return errors.Trace(err)
+			}
+			conf.Security.SSLCertBytes, err = ioutil.ReadFile(conf.Security.CertPath)
+			if err != nil {
+				return errors.Trace(err)
+			}
+			conf.Security.SSLKEYBytes, err = ioutil.ReadFile(conf.Security.KeyPath)
+			if err != nil {
+				return errors.Trace(err)
+			}
+		}
+		tlsConfig, err = utils.ToTLSConfigWithVerifyByRawbytes(conf.Security.SSLCABytes,
+			conf.Security.SSLCertBytes, conf.Security.SSLKEYBytes, []string{})
 		if err != nil {
 			return errors.Trace(err)
 		}
+		// NOTE for local test(use a self-signed or invalid certificate), we don't need to check CA file.
+		// see more here https://github.com/go-sql-driver/mysql#tls
+		if conf.Host == "127.0.0.1" {
+			tlsConfig.InsecureSkipVerify = true
+		}
```

Comment:
Why ignore this here? Do we have examples in other programs?

Reply:
The self-signed certificate generated by `mysql_ssl_setup` does not contain 127.0.0.1

Then the go-mysql documentation also recommends skipping the ca check in this case

The example will be in the DM test